ani-cli

```diff
@@ -413,7 +413,11 @@ open_episode () {
 		sed -E "
 			s/^${selection_id}\t[0-9]+/${selection_id}\t$((episode+1))/
 		" "$logfile" > "${logfile}.new"
-		kill "$PID" >/dev/null 2>&1
+
+        if [ "$PID" != "" ] && [ "$auto_play" -eq 0 ]; then
```

Comment:
I would personally use `-z` instead of `!= ""`, much cleaner. Also, there's no reason for the empty lines, please remove them. The lines should be indented one further, since they are still part of the previous `if ` branch. This will be solved if you replace your spaces with tabs.